src/main/java/com/project/videotecha/exception/EmailNotSentException.java

```diff
@@ -0,0 +1,8 @@
+package com.project.videotecha.exception;
+
+public class EmailNotSentException extends RuntimeException {
+
+    public EmailNotSentException(String message) {
+        super(message);
+    }
```

Comment:
Blank line before the end of the class, because there is a blank line at the beginning